content/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork.md

```diff
@@ -19,6 +19,12 @@ topics:
 
 ## Syncing a fork from the web UI
 
+{% note %}
+
+**Note**: You can only sync a fork if you have write permissions
+
+{% endnote %}
```

Comment:
```suggestion
```
Rather than having a note here, we should add a `permissions:` element to the front matter. 

We don't usually use callouts like `{% note %}` for general information such as permissions or prerequisites.

So in the front matter, underneath the `intro:` we should add something like

```
permissions: 'People with write access for a forked repository can sync the fork to the upstream repository.'
```

References: https://github.com/github/docs/blob/main/contributing/content-model.md#permissions-statements, https://github.com/github/docs/blob/main/contributing/content-style-guide.md#permissions, https://github.com/github/docs/tree/main/content#permissions

Reply:
Hey @ethanpalm I have removed the note and shifted it to the permissions frontmatter.